website/src/_includes/section-menu/contribute.html

```diff
@@ -12,12 +12,11 @@
 
 <li><span class="section-nav-list-main-title">Contribute</span></li>
 <li><a href="{{ site.baseurl }}/contribute/">Get started contributing</a></li>
-<li><a href="https://cwiki.apache.org/confluence/display/BEAM/Development+Environment+FAQ">FAQ</a></li>
+<li><a href="{{ site.baseurl }}/contribute/get-help/">Get Help</a></li>
```

Comment:
I think we should have a direct link specifically for "development setup" or equivalent wording. This is something many look for and not necessarily expect to find on a general help page.

Reply:
Is this covered by the material in "Get Started Contributing"? Otherwise I don't know what it means, really. I assume that anyone interested in contributing already has a preferred development set up.

Reply:
Yes, it would fit under "Get Started Contributing". Probably it should appear as one of the first things under "Prerequisites for contributing code". Folks interested contributing would look for how to setup their environment.